.env

```diff
@@ -0,0 +1,3 @@
+DB_URL=postgres://team3:1234@localhost:5432/todo_db
+TEST_DB_URL=postgres://team3:1234@localhost:5432/todo_test_db
+SECRET_KEY=whatDoYouWantMeToMakeHereSoThatItBcomesReadable
```

Comment:
Probably you got this pushed to GitHub because it was called `.env` then at some pull request it was changes to `config.env` and so `.env` was removed from the `.gitignore` file. 
See, this is the importance of well documenting the PR.